README.rst

```diff
@@ -692,6 +693,8 @@ to the default list of symfony-level fixers:
         ->finder($finder)
     ;
 
+NOTE: `exclude` will work only for directories, so if you need to exclude file, try `notPath`.
```

Comment:
symfony docs are hard to read, they are written probably in api point of view not actual real life examples, like here. and it also does not emphasis that `exclude` **does not work** for files.